CHANGELOG.md

```diff
@@ -7,12 +7,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [2.1.1] - 2025-07-09
-
-### Changed
-
-- Increased `oidc.expiry.refreshTokens.validIfNotUsedFor` to 30 days, according to `oidc.expiry.refreshTokens.absoluteLifetime`
-
 ## [2.1.0] - 2025-07-03
```

Comment:
Looks like this changelog got lost with merging main. Please make sure the existing entries stay, and your new changelog is added.